bounded-core/src/main/scala/io/cafienne/bounded/akka/persistence/eventmaterializers/ReadJournalOffsetStore.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2016-2018 Cafienne B.V. <https://www.cafienne.io/bounded>
+ */
+
+package io.cafienne.bounded.akka.persistence.eventmaterializers
+
+import java.util.concurrent.TimeUnit
+
+import akka.persistence.cassandra.query.scaladsl.CassandraReadJournal
+import akka.persistence.query.Offset
+import io.cafienne.bounded.akka.ActorSystemProvider
+import io.cafienne.bounded.akka.persistence.ReadJournalProvider
+import io.cafienne.bounded.akka.persistence.eventmaterializers.offsetstores.{
+  CassandraOffsetStore,
+  InMemoryBasedOffsetStore,
+  OffsetStore
+}
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+trait ReadJournalOffsetStore extends OffsetStore {
```

Comment:
Do we still need the ReadJournalOffset store ? 
If so, I think that when leveldb is chosen, the offset store should be given for lmdb.